support/scripts/checkpatch.uk

```diff
@@ -27,7 +27,9 @@ fi
 
 _CHECKPATCHCONF=
 if [ -f "${_SELF_BASE}/../../.checkpatch.conf" ]; then
+#	readarray -t _CHECKPATCHCONF -d ' ' < "${_SELF_BASE}/../../.checkpatch.conf"
 	_CHECKPATCHCONF=$(cat "${_SELF_BASE}/../../.checkpatch.conf")
 fi
 
-exec "${_CHECKPATCHPL}" "${_CHECKPATCHCONF}" "$@"
+# shellcheck disable=SC2086
```

Comment:
It would be a good practice in general to also leave a brief comment with reasoning, but then again that's why we have `git blame` for. So up to you 🤟🏼

Reply:
I'll leave one, no worries

Reply:
Funny, just finding this PR now: I opened a similar one, trying to fix the same thing: PR #1276.